src/httpd_jsonapi.c

```diff
@@ -31,6 +31,7 @@
 #endif
 
 #include <regex.h>
+#include <sqlite3.h>
```

Comment:
Can't have direct dependencies to sqlite, db.c must serve as an abstraction (with the exception of cache.c). I also don't actually see anything below that needs sqlite3.h included?